src/libraries/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -1043,6 +1073,273 @@ public static void Xml_DeserializeEmptyDateTimeOffsetType()
         }
     }
 
+    [Theory]
+    [InlineData("0001-1-1")]
+    [InlineData("2002-6-17")]
```

Comment:
Should these be in ISO8601 correct format, ie leading zeros for single digit numbers. The DateOnly.Parse method hides this as it's very lax in parsing the dates. Also is calling just Parse without any format specifiers going to be robust to different cultures being set? I presume the leading year would force it to interpret it as YYYY-M-d, but I'm not certain on that. I would be more comfortable with an explicit parsing custom format string to protect against different cultures.

Reply:
The data is only used directly in the test code to generate our source `DateOnly`. So it's not important for these strings to match the perscribed formats used by the serializer. However, some testcases are doing string comparison with the input string. So, in the name of consistency, I'll update this.